cli/network.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/alibaba/pouch/apis/types"
+
+	"github.com/spf13/cobra"
+)
+
+// networkDescription defines the network command description and auto generate command doc.
+var networkDescription = "Manager the networks in pouchd. " +
+	"It contains the functions of create/remove/list/inspect network, 'driver' is used to list drivers that pouch support. " +
+	"Now bridge network is supported in pouchd defaulted, it will be initialized when pouchd starting."
+
+// NetworkCommand is used to implement 'network' command.
+type NetworkCommand struct {
+	baseCommand
+}
+
+// Init initializes NetworkCommand command.
+func (n *NetworkCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "network [command]",
+		Short: "Manage pouch networks",
+		Long:  networkDescription,
+		Args:  cobra.MinimumNArgs(1),
+	}
+
+	c.AddCommand(n, &NetworkCreateCommand{})
+}
+
+// networkCreateDescription defines the comand network create description.
```

Comment:
typo 'comand'->'command', but I think the comment like `networkCreateDescription is used to describe network create command in detail and auto generate command doc.` which is consistent with comments in volumes.go is much better.

Reply:
ok